tests/khmer_tst_utils.py

```diff
@@ -53,7 +53,27 @@ def cleanup():
     cleanup_list = []
 
 
+def scriptpath(scriptname='interleave-reads.py'):
+    "Return the path to the scripts, in both dev and install situations."
+
+    # note - it doesn't matter what the scriptname is here, as long as
+    # it's some khmer script present in this version of khmer.
+
+    path = os.path.join(os.path.dirname(__file__), "../scripts")
+
+    if os.path.exists(os.path.join(path, scriptname)):
+        return path
+
+    for path in os.environ['PATH'].split(':'):
+        if os.path.exists(os.path.join(path, scriptname)):
```

Comment:
> The difference is that now we are consulting the path for all script running not just the piped streaming tests.

Uh, no, I don't think so?

https://github.com/dib-lab/khmer/blob/master/tests/khmer_tst_utils.py#L71

(All of this is just refactoring, nothing new.)